src/classes/MyHandler/MyHandler.ts

```diff
@@ -880,70 +880,48 @@ export default class MyHandler extends Handler {
 
         const offerMessage = offer.message.toLowerCase();
 
-        if (itemsToGiveCount === 0) {
-            const isGift = [
-                'gift',
-                'donat', // So that 'donate' or 'donation' will also be accepted
-                'tip', // All others are synonyms
-                'tribute',
-                'souvenir',
-                'favor',
-                'giveaway',
-                'bonus',
-                'grant',
-                'bounty',
-                'present',
-                'contribution',
-                'award',
-                'nice', // Up until here actually
-                'happy', // All below people might also use
-                'thank',
-                'goo', // For 'good', 'goodie' or anything else
-                'awesome',
-                'rep',
-                'joy',
-                'cute', // right?
-                'enjoy',
-                'prize',
-                'free',
-                'tnx',
-                'ty',
-                'love',
-                '<3'
-            ].some(word => offerMessage.includes(word));
-
-            if (isGift) {
-                // We can accept escrow if it's gift
-                if (checkBannedFailed) {
-                    offer.log('info', `is a gift offer, but failed to check for banned status, declining...`);
-                    return {
-                        action: 'decline',
-                        reason: 'GIFT_FAILED_CHECK_BANNED',
-                        meta: isContainsHighValue ? { highValue: highValueMeta } : undefined
-                    };
-                }
-
-                offer.log(
-                    'trade',
-                    `is a gift offer, accepting. Summary:\n${JSON.stringify(
-                        summarize(offer, this.bot, 'summary-accepting', false),
-                        null,
-                        4
-                    )}`
-                );
+        const forcesReview = offerMessage === 'force-review'; // maybe there is a better keyword?
```

Comment:
I think it's better to make it not be the exact keyword (use `Array.some` on the list of possible keywords).
Possible keywords:
- `refund`
- `review`
- `check`